release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -872,6 +887,10 @@ Support for empty files using the `--registry-config` and `--to` flags in `oc re
 [id="ocp-4-10-removed-features"]
 === Removed features
 
+{product-title} 4.10 removes the Jenkins Operator, which was a Technology Preview feature, from the OperatorHub page in the {product-title} web console interface. Bug fixes and support are no longer available.
```

Comment:
```suggestion
{product-title} 4.10 removes the Jenkins Operator, which was a Technology Preview feature, from the *OperatorHub* page in the {product-title} web console interface. Bug fixes and support are no longer available.
```
- Per the SSG,  names of UI elements need to be in bold text. I'm not sure if `OperatorHub` is a UI element (i.e. name of the page that appears in the UI), though.

Reply:
Updated. Thanks.